docs/deploy-coe-with-Kafka.md

```diff
@@ -86,7 +86,7 @@ To edit the YAML file for the required changes, perform the following steps:
         {
             "Endpoints": {
               "KAFKA": {
-                "ServerUrl": "X.X.X.X:9092", #Specify the Kafka broker IP
+                "ServerUrl": "X.X.X.X:9092,Y.Y.Y.Y:9092", #Specify the Kafka broker IP
```

Comment:
```suggestion
                "ServerUrl": "X.X.X.X:9092,Y.Y.Y.Y:9092", #Specify the bootstrap Kafka broker addresses (separated by comma in case multiple bootstrap brokers are to be configured)
```]